### ISSUE TITLE
`i.nhentai.net` Image Server Not Working

### ISSUE THREAD

**Issue by getBoolean** [AlexandreSenpai/Enma#86]
Images uploaded on nhentai aren't able to be loaded from the `i.nhentai.net` image domain anymore. Instead there are multiple image servers (1-4), e.g., `i1.nhentai.net`.

Test Image: `https://i1.nhentai.net/galleries/3156215/1.webp` (image not explicit)

I don't know which of the 4 image servers Enma should be using, it probably depends on region. Perhaps just using `i1` would be safest.

**Comment by AlexandreSenpai** [comment]
Thank you very much for spotting that. I'm going to check this behaviour and come up with some changes

**Comment by AlexandreSenpai** [comment]
Feel free to suggest any changes if you feel like it 🫡

**Comment by AlexandreSenpai** [comment]
Fixed on version 2.4.4!!

Now you have the power of customizing NHentai default parameters

```python
from typing import cast
from enma import Enma, Sources
from enma import logger, LogMode
from enma import (
    NHentai, 
    NHentaiConfig, 
    NhentaiSourceConfig
)

enma = Enma()
enma.source_manager.set_source(source_name=Sources.NHENTAI)

nh = cast(NHentai, enma.source_manager.source)
nh.set_config(
    config=NHentaiConfig(
        source=NhentaiSourceConfig(
            tiny_image_server="t1",
        )
    )
)

manga = enma.get(identifier="354303", with_symbolic_links=False)
print(manga)
```
